source/extensions/filters/http/cache/cache_headers_utils.h

```diff
@@ -128,16 +124,24 @@ class VaryHeader {
   // Parses the header names that are in the Vary header value.
   static std::vector<std::string> parseHeaderValue(const Http::HeaderEntry* vary_header);
 
+  // Parses the allowlist from the Cache Config into the object's private allowlist_.
+  VaryHeader(const Protobuf::RepeatedPtrField<envoy::type::matcher::v3::StringMatcher>& allowlist);
```

Comment:
nit: s/allowlist/allow_list/g

Reply:
I'm not familiar with the `/g` here, could you explain what it means?

Reply:
it is 'sed' syntax borrowed for code reviews; means do this globally.